Src/BootCamp.Chapter/PersonAndBalance.cs

```diff
@@ -0,0 +1,55 @@
+﻿namespace BootCamp.Chapter
+{
+    public class PersonAndBalance
+    {
+        private string[] _personAndBalanceArray;
+        private float[] _balance;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make '_balance' 'readonly'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5070129)